openedx/core/djangoapps/video_pipeline/tests/mixins.py

```diff
@@ -12,6 +14,17 @@ class VideoPipelineIntegrationMixin(object):
         'enabled': True,
         'api_url': 'https://video-pipeline.example.com/api/v1/',
         'service_username': 'cms_video_pipeline_service_user',
+        'client_name': 'video_pipeline'
+    }
+
+    video_pipelien_oauth_client_defaults = {
+        'name': 'video_pipeline',
+        'url': 'https://video-pipeline.example.com/api/v1/',
+        'redirect_uri': 'https://video-pipeline.example.com/api/v1/',
+        'logout_uri': 'https://video-pipeline.example.com/api/v1/',
```

Comment:
what about having different urls for each like `'logout_uri': 'https://video-pipeline.example.com/api/v1/logout',`